lib/helpers.js

```diff
@@ -0,0 +1,7 @@
+import {
+  assign,
+} from 'lodash';
+
+export function enhance(xs = [], source = {}) {
+  return xs.map(x => assign(source, x));
+}
```

Comment:
I keep defining little utility functions like https://github.com/artsy/metaphysics/blob/8f8ff3709b4ae6544dc1eb962d25bbf853bb4e88/schema/image/normalize.js#L16. This is gonna be my utility belt?